pkg/operator/loglevel/logging_controller.go

```diff
@@ -4,19 +4,26 @@ import (
 	"context"
 
 	operatorv1 "github.com/openshift/api/operator/v1"
-
 	"github.com/openshift/library-go/pkg/controller/factory"
 	"github.com/openshift/library-go/pkg/operator/events"
 	operatorv1helpers "github.com/openshift/library-go/pkg/operator/v1helpers"
 )
 
 type LogLevelController struct {
 	operatorClient operatorv1helpers.OperatorClient
+
+	// for unit tests only
+	setVerbosityFn    func(operatorv1.LogLevel) error
+	currentLogLevelFn func() (operatorv1.LogLevel, bool)
 }
 
 // sets the klog level based on desired state
 func NewClusterOperatorLoggingController(operatorClient operatorv1helpers.OperatorClient, recorder events.Recorder) factory.Controller {
-	c := &LogLevelController{operatorClient: operatorClient}
+	c := &LogLevelController{
+		operatorClient:    operatorClient,
+		setVerbosityFn:    SetVerbosityValue,
+		currentLogLevelFn: CurrentLogLevel,
```

Comment:
why do we have three terms for the same thing? verbosity, verbosityValue, logLevel

Reply:
we can pick one and stick to it, I can unify them

Reply:
yes please